src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -1061,29 +1061,123 @@ public static unsafe void MulElementWiseU(ReadOnlySpan<float> src1, ReadOnlySpan
             }
         }
 
-        public static unsafe float SumU(ReadOnlySpan<float> src)
+        public static unsafe float Sum(ReadOnlySpan<float> src)
```

Comment:
Assuming you're running on a machine supporting AVX -- unit tests would not hit this -- unless you ran them with the env variable set?

Reply:
I assume not since @fiigii change didn't go in yet.

Reply:
We have unit/perf tests that explicitly call these methods/code-paths